.github/python-nosetests.sh

```diff
@@ -6,5 +6,8 @@ set -uex
 sudo add-apt-repository -y "deb http://archive.ubuntu.com/ubuntu/ trusty main universe"
 sudo apt-get update
 sudo apt-get install -y python-mock python-nose
+curl https://bootstrap.pypa.io/pip/2.7/get-pip.py |sudo python2
```

Comment:
If I remember correctly this had to be done because ubuntu doesn't ship and adequate version for pip compatible with python 2.7.

As for the danger, if the runners have permissions to commit to the repo or modify history and pypa.io would be hijacked, we would be aware of the change, unless the SHA1 hashes somehow get bypassed somehow.